R-package/R/lgb.model.dt.tree.R

```diff
@@ -111,7 +111,10 @@ single.tree.parse <- function(lgb_tree) {
         , leaf_count = integer(0L)
       )
       # start tree traversal
-      pre_order_traversal(env, tree_node_leaf, current_depth, parent_index)
+      pre_order_traversal(env = env
+                          , tree_node_leaf = tree_node_leaf
+                          , current_depth = current_depth
+                          , parent_index = parent_index)
```

Comment:
```suggestion
      pre_order_traversal(
          env = env
          , tree_node_leaf = tree_node_leaf
          , current_depth = current_depth
          , parent_index = parent_index
      )
```

Can you please use this indentation to match the rest of the package?

Reply:
OK!

Reply:
thanks! I saw you pushed a commit...in the future, you might find it easier to click the "Commit suggestion" button in the UI

![image](https://user-images.githubusercontent.com/7608904/100493937-fe883600-3101-11eb-842f-11d28a1b78d2.png)

Reply:
Oh, OK, didn't know about that!